gitops/base-install/ebs-csi-driver/install.yaml

```diff
@@ -17,12 +17,31 @@ metadata:
     - resources-finalizer.argocd.argoproj.io
 spec:
   project: admins
-  source:
-    chart: aws-ebs-csi-driver
+  sources:
+  - repoURL: 'https://github.com/pelotech/foundation'
```

Comment:
With my version the CRDs were loaded as part of the Kustomization of the ArgoCD Application, so no sequencing issue AFAIK. 

What does this gain @josmo? That the CRDs and snapshot controller are "within" the `ebs-csi-driver` ArgoCD Application?

Reply:
we chatted about this but I'll add notes for posterities sake
* this route makes all the resources part of the ebs-csi argocd application, also moved the service account to the same pattern
* docs says snapshot controller needs to be running before the driver (I've seen mixed blogs on this) so having the waves from argo (by default is per application) means it's guaranteed that the snapshot controller is up before the drive